python/ray/_raylet.pyx

```diff
@@ -900,28 +901,32 @@ cdef class CoreWorker:
                      args,
                      uint64_t max_reconstructions,
                      resources,
-                     placement_resources):
+                     placement_resources,
+                     is_persistent):
```

Comment:
`is_detached`?

Reply:
You can actually say `c_bool is_detached` here and get rid of the `c_is_persistent` variable below.